test/math.ts

```diff
@@ -223,8 +223,7 @@ testSingleExpressionByList(toFixed_list );
 test("Test that string concatenation is still working after NaN change.", (assert) => {
   let expected = {
     insert: [
-      ["a", "tag", "div"],
-      ["a", "text", "Test Testy"],
+      ["a", "result", "Test Testy"],
```

Comment:
Not especially important, but what's the rationale for choosing `result` over the existing `text`?